processing.tex

```diff
@@ -1,5 +1,127 @@
+  This issue also uncovered the need for more frequent heartbeat message logging for long-running forcedPhotCoadd (DM-33854) and faro tasks (DM-33820).
+  The timeout limit was set again to 20 hours to allow step3 production to continue efficiently.
+
+  \item
+  About once every two workflows or so, a deblend job or two failed due to out-of-memory error (DM-33690), because of very bright/large objects on the coadd image.
+  These failures were fixed in rescue workflows, but required long run times ($>$10 hrs) and large memory ($\approx$~40 GB) using the \texttt{extra-highmem} queue, though ultimately with deblending results that were likely unreliable.
+  The most extreme example (tract 4648, patch 29) took 12 unsuccessful attempts before finally succeeding in the \texttt{extra-highmem-non-preempt} queue (DM-33947), after 2 days 22.7 hours run time, 190 GB memory, and extra efforts from the PanDA team to bypass heartbeat logging issues.
+  This was the last image processing job to be completed in step3.
+  DM-33690 implemented deblending configuration changes to skip these problematic deblends, so this should not be an issue subsequent to DP0.2.
+
+  \item
+  The large faro tasks matchCatalogsPatchMultiBand and especially matchCatalogsTract took long run times and large memory ($\approx$~130 GB for the latter).
+  They needed to be run on the \texttt{extra-highmem} queue and were also prone to preemption and multiple re-attempts due to heartbeat logging issues (DM-33820).
+
+ \item
+  Some delays arose when jobs became stuck as they were unexpectedly scheduled to a non-IDF queue, which had its ``region'' set erroneously to ``LSST'', same as for the IDF queues.
+  This issue was corrected by the PanDA support team.
+
+  \item
+  There were also some delays due to unexpected consequences from PanDA server and client updates, again resolved by the PanDA support team.
+
+\end{itemize} %step3
+
+\item step4
+\begin{itemize}
+
+  \item
+  An attempt was made to use the Google Artifact Registry (GAR) instead of Docker Hub to download the LSST processing stack.
+  However, this resulted in intermittent authentication problems that led to 20-50\% job failures in three step4 workflows.
+  Despite an attempt to reconfigure the IDF computing cluster involved, the authentication problems continued, and we reverted to using Docker Hub and reran the affected workflows.
+
+  \item
+  We also saw a drop in the maximum number of running jobs in the IDF queue involved, from nearly 4000 down to as few as about 2000, possibly related to the above authentication issues.
+  PanDA support increased the maximum number of new worker nodes created from
+50 to 80 to help stabilize the number of running jobs back to nearly 4000.
+
+
+\end{itemize} %step4
+
+\item step5
+\begin{itemize}
+
+  \item
+  We encountered a very slow start to jobs running on the PanDA system due to the large number of jobs per task ($>$100,000) in the first step5 workflow.
```

Comment:
At some point we needed to ask for an iDDS server with more resources and they also made it more efficient. I think we have been running the largest DAGs iDDS had had to handle.

Reply:
added a line